change/@fluentui-react-tooltip-dafa57b4-c8b3-47b0-8790-bb85041f64e2.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "fix(Tooltip): Add transparent outline to arrow for HC",
```

```suggestion
  "comment": "Add transparent outline to arrow for HC",
```

The same thing. As it goes to change under `react-tooltip` I don't see need in it